### PR TITLE
Legacy fix 1140

### PR DESCRIPTION
**Pull request by dannychapman**
- Updated IA as requested by @bethpotts 
- Completes #1140
- Pulled Modernizr out of webpack, as it is not running locally. 
- Removed animate.js library to improve site performance

Re: webpack changes, please review @awong-dev 

Thanks!

### CODE REVIEW

**Comment by awong-dev** [comment]
@dannychapman what's going on with modernizr and webpack? I'd rather not revert the config.

**Comment by dannychapman** [comment]
@awong-dev for some reason (can't figure out why), Modernizr is not firing. It does browser feature detection and adds css classes to the html element. When I pull it out and load separately, it works fine.

**Comment by awong-dev** [comment]
@dannychapman Right...I tested modernizr locally here on chrome and it's working in the macro sense (window.Modernizer is defined, and various feature detects exist).

However, with webpack, we're not loading the JS you've specified. Instead, I'm building modernizer from scratch using the modernizer generator (see .modernizrrc). It's possible I have the wrong build configs for modernizr and left out the "css classes" feature?

What browser are you testing on, and what DOM attribute are we missing

**Comment by dannychapman** [comment]
@awong-dev ah, that could be it. I made my own very stripped down custom build of Modernizr. All I'm using are:

- csstransitions 
- touchevents

**Comment by awong-dev** [comment]
Here's the config I reverse engineered from your build:

```
{
  "options": [
    "html5shiv",
    "prefixed",
    "setClasses"
  ],
  "feature-detects": [
    "svg",
    "svg/inline",
    "svg/clippaths",
    "touchevents",
    "css/mediaqueries",
    "css/transitions",
    "dom/classlist",
    "dom/dataset",
    "ie8compat"
  ]
}
```

I didn't quite know what to exercise to test the usage though.

**Comment by dannychapman** [comment]
@awong-dev anything you can add for touch events? That's what I'm using in the footer to load the correct footer-crisis box for touch or no touch (touch dials the number)

File: _includes/header.html [hunk 1]
```diff
@@ -78,6 +78,11 @@
 <script src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=VA" id="_fed_an_ua_tag"></script>
 <script src="/assets/js/generated/bundle.js"></script>
 
+<!--[if lt IE 9]>
```
Comment (awong-dev) [inline _includes/header.html]: This shouldn't be needed anymore I don't think.

Though I'm not sure how to test...

**Pull request merged**